crates/rpc/rpc/src/otterscan.rs

```diff
@@ -0,0 +1,108 @@
+#![allow(dead_code, unused_variables)]
+use crate::result::internal_rpc_err;
+use async_trait::async_trait;
+use jsonrpsee::core::RpcResult;
+use reth_primitives::{Address, BlockId, TxHash, H256, U256};
+use reth_rpc_api::{EthApiServer, OtterscanServer};
+use reth_rpc_types::{
+    BlockDetails, ContractCreator, InternalOperation, TraceEntry, Transaction,
+    TransactionsWithReceipts,
+};
+
+/// Otterscan Api
+#[derive(Debug)]
+pub struct OtterscanApi<Eth> {
+    eth: Eth,
+}
+
+impl<Eth> OtterscanApi<Eth> {
+    /// Creates a new instance of `Otterscan`.
+    pub fn new(eth: Eth) -> Self {
+        Self { eth }
+    }
+}
+
+#[async_trait]
+impl<Eth> OtterscanServer for OtterscanApi<Eth>
+where
+    Eth: EthApiServer,
+{
+    /// Handler for `ots_hasCode`
+    async fn has_code(&self, address: Address, block_number: Option<BlockId>) -> RpcResult<bool> {
+        Err(internal_rpc_err("unimplemented"))
+    }
+
+    /// Handler for `ots_getApiLevel`
+    async fn get_api_level(&self) -> RpcResult<u64> {
+        Err(internal_rpc_err("unimplmented"))
```

Comment:
```suggestion
        Err(internal_rpc_err("unimplemented"))
```

Couple more down below, might want to search/replace `unimplmented` -> `unimplemented`.

Reply:
nice catch ! thanks